pkg/common.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkg
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/ossf/scorecard/v2/checker"
+	"go.uber.org/zap/zapcore"
+)
+
+// TODO: Verify this works in GitHub's dashboard.
+func textToHTML(s string) string {
+	return strings.ReplaceAll(s, "\n", "<br>")
```

Comment:
Why do we need this? Because a single \n doesn't translate to \n in rendered markdown?

How about replacing the single newline with double?

Reply:
good point. So it's not clear to me how GitHub handles this yet. But I can change to double space, for sure. I'm hoping they refuse html content when displaying results :-)